pylops_mpi/basicoperators/__init__.py

```diff
@@ -0,0 +1 @@
+from .BlockDiag import MPIBlockDiag
```

Comment:
I suggest to follow the structure of pylops https://github.com/PyLops/pylops/blob/dev/pylops/basicoperators/__init__.py

Reply:
Done...

Reply:
Great :)